adapters/adyoulike/usersync.go

```diff
@@ -0,0 +1,12 @@
+package adyoulike
+
+import (
+	"text/template"
+
+	"github.com/prebid/prebid-server/adapters"
+	"github.com/prebid/prebid-server/usersync"
+)
+
+func NewAdyoulikeSyncer(urlTemplate *template.Template) usersync.Usersyncer {
+	return adapters.NewSyncer("adyoulike", 0, urlTemplate, adapters.SyncTypeRedirect)
```

Comment:
Aren't you registered with the GVL with vendor is 259?

Reply:
Yes indeed, thank you.